### PR TITLE
:rocket: Send logs to Splunk in JSON

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-redirect succeeded (http://gp-redirect-pr-33.dev.beta.nhschoices.net)

**Review by adamgraves-choices — APPROVED** [review]

File: rancher-config/docker-compose.yml [hunk 1]
```diff
@@ -13,3 +13,14 @@ services:
...
       io.rancher.container.pull_image: always
       io.rancher.scheduler.affinity:host_label_soft: c2s=true
+    logging:
+      driver: splunk
+      options:
+        splunk-url: ${SPLUNK_HEC_URL}
+        splunk-token: ${SPLUNK_HEC_TOKEN}
+        splunk-insecureskipverify: "true"
+        splunk-sourcetype: docker
+        splunk-source: profiles
```
Comment (adamgraves-choices) [inline rancher-config/docker-compose.yml]: This needs to be `gp-redirect`

**Review by adamgraves-choices — CHANGES_REQUESTED** [review]

File: rancher-config/docker-compose.yml (hunk 1 above)
Comment (st3v3nhunt) [reply rancher-config/docker-compose.yml]: Updated

**Review by adamgraves-choices — APPROVED** [review]

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/gp-redirect succeeded (http://gp-redirect-pr-33.dev.beta.nhschoices.net)

**Pull request merged**